providers/common/include/prov/securitycheck.h

```diff
@@ -29,3 +34,9 @@ int ossl_digest_get_approved_nid(const EVP_MD *md);
 int ossl_digest_rsa_sign_get_md_nid(const EVP_MD *md);
 int ossl_securitycheck_enabled(OSSL_LIB_CTX *libctx);
 int ossl_tls1_prf_ems_check_enabled(OSSL_LIB_CTX *libctx);
+int ossl_hkdf_digest_check_enabled(OSSL_LIB_CTX *libctx);
```

Comment:
same comment as other PR. these are not really needed since the code that uses it is #ifdef FIPS_MODULE

Reply:
Removed.